### ISSUE TITLE
4071 Section 2.2

### ISSUE THREAD

**Issue by jlivingood** [IASA2/draft-ietf-iasa2-struct#11]
2.2 Principles of the IASA, IETF, and ISOC Relationship 

This can point to the new ISOC-IETF Relationship doc in process w/Gonzalo (to be published soon in -00).

**Comment by josephlhall** [comment]
going to leave this one open until that -00 hits the streets.

**Comment by josephlhall** [comment]
Any update on this doc from Gonzalo?

**Comment by jlivingood** [comment]
Yes – just posted at https://datatracker.ietf.org/doc/draft-ietf-iasa2-rfc2031bis/

**Comment by josephlhall** [comment]
I've pointed to this in the document but we might want to summarize it in a few sentences in 4071bis too. I'll leave it as a TODO and we can close out when we think it's right.
